### PR TITLE
Update plugin.json

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated punctuation for UI consistency.

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
         "settings": [{
             "key": "CustomAttributes",
             "type": "custom", 
-            "display_name": "Custom Attributes"
+            "display_name": "Custom Attributes:"
```
Comment (marianunez) [inline plugin.json:24]: I wonder if the product should add this instead of having to expect all plugins to add the colon to be consistent?

Comment (justinegeffen) [reply plugin.json:24]: Would having this in a style guide/in the app development workflow be helpful at all, or should it be more automated? And who would be the best person to do the latter? :)

Comment (jfrerich) [reply plugin.json:24]: @marianunez, this is a good catch and proposal.  Probably a UX consideration also?  @abhijit-singh?

**Review by marianunez — APPROVED** [review]
Approving to unblock @justinegeffen but I think this would be a simple ticket for Integrations to include the `:` if not already provided in the `display_name` of the plugin in order to keep consistency across how the labels should display in the System Console.

**Review by jfrerich — APPROVED** [review]
I agree with Maria.  Would you mind creating a ticket?

**Comment by marianunez** [comment]
> I agree with Maria. Would you mind creating a ticket?

Created at https://mattermost.atlassian.net/browse/MM-32355

**Pull request merged**